pkg/action/action.go

```diff
@@ -255,6 +257,17 @@ func (c *Configuration) Init(getter genericclioptions.RESTClientGetter, namespac
 		}
 		d.SetNamespace(namespace)
 		store = storage.Init(d)
+	case "sql":
```

Comment:
I know this is the way it worked on Helm 2... but maybe we could just set this (`HELM_DRIVER`) to "postgres" and then get rid of `HELM_DRIVER_SQL_DIALECT`.